lib/arel/visitors/oracle.rb

```diff
@@ -31,9 +31,9 @@ def visit_Arel_Nodes_SelectStatement o, collector
           if offset.expr.is_a? Nodes::BindParam
             offset_bind = nil
             collector << ') raw_sql_ WHERE rownum <= ('
-            collector.add_bind(offset.expr) { |i| offset_bind = ":a#{i}" }
+            collector.add_bind(offset.expr.value) { |i| offset_bind = ":a#{i}" }
```

Comment:
Thanks for the review. This is one of the dirty hacks :cry:  . Then I do not know how to implement visitor pattern here. Are you suggesting replacing this line to `visit offset.expr`?